### ISSUE TITLE
How to run installed tools?

### ISSUE THREAD

**Issue by cszatmary** [bwplotka/bingo#52]
After I install tools with `bingo get` what is the intended way to run them?

It would be nice if there was a command where given a tool name it would find the path to the installed tool and execute it.

Example of what this might look like:
```
bingo run golangci-lint run
```

**Comment by bwplotka** [comment]
Hey, sorry for the delayed response, I am lagging behind GH notifications 😱 

This is a valid question overall. I think it was not well explained. I improved docs here: https://github.com/bwplotka/bingo/pull/56 @cszatmary does it make sense? 🤗 

There are some tools that wanted to support something like `run` command, but I believe there is no need for something like this. We can totally use native OS tools like `shell` and [its `variable`](https://www.tutorialspoint.com/unix/unix-using-variables.htm) to achieve the same, no? See [this doc](https://github.com/bwplotka/bingo/pull/56/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R80).

To sum up, I see 3 reasons why using the native way might be better:

* We don't reproduce work / reinvent the wheel
* Supporting `run` well actually is not that easy to write. Passing arguments by CLI to underlying shell execution is actually incredibly hard (trust me, I tried in my previous job). There are thousands of nuances (e.g how to differentiate between `bingo` flags and passed flags? `--` is the native way, but it also means you cannot use the same commands anymore.)
* Would nice to maintain in bingo [do one thing, and do it well] philosophy (https://hackaday.com/2018/09/10/doing-one-thing-well-the-unix-philosophy/)

Hope this makes sense (:

**Comment by wmcnamee-coreweave** [comment]
ya, I'm in the same boat here. I just found out about this tool, which seems good, except it would be great if symlinks were made instead of having to source the `variables.env` and use all caps `TOOL_NAME`. I essentially need tool installs to be transparent.
